ext/web/lib.deno_web.d.ts

```diff
@@ -140,11 +140,22 @@ declare type EventListenerOrEventListenerObject =
   | EventListenerObject;
 
 interface AddEventListenerOptions extends EventListenerOptions {
+  /** A boolean value indicating that the listener should be invoked at most
+   * once after being added. If true, the listener would be automatically
```

Comment:
I sort of disagree with MDN's grammar here. I think it should be `"If true, the listener **will** be automatically removed when invoked.` Also, perhaps `after invoked`. But MDN is a pretty good source to use.